src/TwigComponent/doc/index.rst

```diff
@@ -72,6 +94,8 @@ file to control the template directory for your components:
             # Namespace & directory for components
             App\Twig\Components\: 'components/'
 
+That's it! We're ready to go!
```

Comment:
I prefer the way it was before. For most of new users, composer require will be enough.